app/controllers/patients_controller.rb

```diff
@@ -803,7 +803,15 @@ def current_case_status
 
   # Fetches table data for viable HoH options.
   def head_of_household_options
-    patients_table_data(params)
+    begin
+      patients = patients_table_data(params, current_user)
+    rescue ActionController::ParameterMissing
```

Comment:
I think this part is not necessary because it will be caught/handled by the `StandardError` rescue